pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -257,5 +258,13 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration, featur
 		allErrors = append(allErrors, fmt.Errorf("invalid configuration: memoryThrottlingFactor %v must be greater than 0 and less than or equal to 1.0", *kc.MemoryThrottlingFactor))
 	}
 
+	if !path.IsAbs(kc.PodLogsPath) {
+		allErrors = append(allErrors, fmt.Errorf("invalid configuration: pod logs path %q must be absolute path", kc.PodLogsPath))
+	}
+
+	if path.Clean(kc.PodLogsPath) != kc.PodLogsPath {
+		allErrors = append(allErrors, fmt.Errorf("invalid configuration: pod logs path %q must be normalized", kc.PodLogsPath))
+	}
```

Comment:
for node reviewers: are there other restrictions we should apply around this filename (no spaces in path, ascii-only characters, etc)?

Reply:
should we forbid a trailing slash?

Reply:
> should we forbid a trailing slash?

No, `filepath.Join` should handle that just fine, unless I'm missing something else here?

Reply:
> for node reviewers: are there other restrictions we should apply around this filename (no spaces in path, ascii-only characters, etc)?

this question is still outstanding

Reply:
since we use this path in metrics, limiting to ascii character is a good idea. We can always expand the charset later

Reply:
this comment still stands. Let's limit this path to ascii.

Reply:
Fixed in https://github.com/kubernetes/kubernetes/pull/112957/commits/4bad8c84991c066d309c52f41e48f0fc20c22229